similar/main/polyobj.cpp

```diff
@@ -569,6 +570,9 @@ void draw_polygon_model(grs_canvas &canvas, const vms_vector &pos, const vms_mat
 
 void free_polygon_models()
 {
+#if defined(DXX_BUILD_DESCENT_II)
+	Exit_models_loaded = 0; // reload exit models again if needed
```

Comment:
Could you explain why this reset is here?  This function only unloads polygon model data, but does not call `bm_unload_last_objbitmaps` or do equivalent work.

Reply:
Because the exit models are still unloaded, and would need to be loaded again. It is fairly safe to assume that unloading models on their own never happens separate from also unloading other game data, which would include the bitmaps. If not, the only side effect is that the exit textures get temporarily doubled in memory.

Reply:
Of course, it would be possible to have two separate flags instead - one for the bitmaps and one for the models.

Reply:
I did separate the two flags - I couldn't find any more spots where Exit_bitmaps_loaded should be reset.